src/extension.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
```

Comment:
o.o some weird issue with the change I did to the `addlicense` make rule ...

```
2022/04/06 22:35:20 -c error: lstat -c: no such file or directory
2022/04/06 22:35:20 Mineiros GmbH error: lstat Mineiros GmbH: no such file or directory
2022/04/06 22:35:20 skipping: .github/workflows/ci.yml
2022/04/06 22:35:20 skipping: .vscode-test/extensions/ms-vscode.js-debug-companion-1.0.17/ci.yml
```
@katcipis

Reply:
WTF ? it is handling the Mineiros GmBH as a filename ? o.O

Reply:
It was the wildcard not being escaped.. fixed.